src/api/app/helpers/webui/scm_exception_messages_helper.rb

```diff
@@ -0,0 +1,90 @@
+module Webui::ScmExceptionMessagesHelper
+  def exception_message(exception:, scm:)
+    case scm
+    when 'GitHub'
+      github_exception_message(exception: exception)
+    when 'GitLab'
+      gitlab_exception_message(exception: exception)
+    end
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Maybe you could split this class into three: `Webui::ScmExceptionMessageHelper`, one for Github errors and another for GitLab errors. Then every vendor has their errors in one class and this class is smaller

Reply:
This is because of the long method. Splitting this into separate classes will still produce this warning because the methods' size will remain the same.